enhancements/windows-containers/health-management.md

```diff
@@ -221,8 +222,12 @@ For Node configuration WICD will have two separate responsibilities:
 
 #### Bootstrap command
 
-When run with the `bootstrap` command, WICD will do the steps required to ensure that Node is created
-for the instance. This functionality is currently implemented within WMCB with the `init-kubelet` command.
+When run with the `bootstrap` command, WICD will do the steps required to ensure that Node is created for the instance. 
+When containerd is the container runtime, the `bootstrap` commnad will have the added responsibility of starting 
+the containerd service. This is becuase the kubelet service has a dependency on the container runtime; the container 
+runtime must have a reachable endpoint when kubelet is initialized. The containerd service will be configured based on 
+the information in the services ConfigMap, pointing to CNI plug-ins' (binaries and config files) locations on the instnace.
```

Comment:
I thought the CNI config gets populated after hybrid-overlay runs. How are you handling that?

Reply:
Correct, but if we point containerd to the _eventual_ location of CNI config, the service will pick it up once created without 
1. erroring out on service start up
2. having to restart 
Unless I am mistaken, this is the behavior @selansen saw in POC work.

Reply:
Please call that out here.